artifacts/LogonScript.ps1

```diff
@@ -496,6 +496,9 @@ Write-Host "Arc enable the kubernetes cluster $ClusterName" -ForegroundColor Cya
 Invoke-WebRequest -Uri https://secure.globalsign.net/cacert/Root-R1.crt -OutFile c:\globalsignR1.crt
 Import-Certificate -FilePath c:\globalsignR1.crt -CertStoreLocation Cert:\LocalMachine\Root
 
+# Temporary fix https://portal.microsofticm.com/imp/v5/incidents/details/564169938/summary
+robocopy "C:\Program Files\AksEdge\kubectl" "$env:userprofile\.azure\kubectl-client" kubectl.exe
+
```

Comment:
`robocopy` output is not captured in the `c:\temp\LogonScript.log` file. 
Do you think we should add a check to confirm the file exists and then `Write-Host "Copied robocopy.exe to $env:userprofile\.azure\kubectl-client"` or similar ?

Reply:
good point